app/assets/javascripts/pretty-text/engines/discourse-markdown/emoji.js.es6

```diff
@@ -7,7 +7,7 @@ let _unicodeRegexp;
 export function setUnicodeReplacements(replacements) {
   _unicodeReplacements = replacements;
   if (replacements) {
-    _unicodeRegexp = new RegExp(Object.keys(replacements).join("|"), "g");
+    _unicodeRegexp = new RegExp(Object.keys(replacements).sort().reverse().join("|"), "g");
```

Comment:
we want the longer emojis sequences to be match first:

👩🏾‍🎓  before 👩🏾‍  before 👩

does it make sense ?